operator/pkg/client/client.go

```diff
@@ -185,11 +187,17 @@ func (c *Client) registerHandlers(socketClient *socket.Client) error {
 		}
 	})
 	if err != nil {
-		return errors.Wrap(err, "prefight handler")
+		return errors.Wrap(err, "preflight handler")
 	}
 
 	err = socketClient.On("deploy", func(h *socket.Channel, args ApplicationManifests) {
 		log.Println("received a deploy request")
+		if args.PreviousManifests != "" {
+			if err := c.diffAndRemovePreviousManifests(args); err != nil {
+				log.Printf("error diffing and removing previous manifests: %s", err.Error())
+				return
```

Comment:
Should this be a fatal error, or should we continue and try to deploy the newer app version anyways?